03-loop.md

```diff
@@ -362,7 +362,7 @@ Sure enough,
 the maxima of these data sets show exactly the same ramp as the first,
 and their minima show the same staircase structure.
 
-> FIXME {.challenge}
+> From 1 to N {.challenge}
```

Comment:
Yes, challenge titles should be level-2 headings.  The validation tool 
should catch this...